numba/np/math/cmathimpl.py

```diff
@@ -0,0 +1,542 @@
+"""
+Implement the cmath module functions.
+"""
+
+
+import cmath
+import math
+
+from numba.core.imputils import Registry, impl_ret_untracked
+from numba.core import types, cgutils
+from numba.core.typing import signature
+from numba.cpython import builtins, mathimpl
+from numba.core.extending import overload
+
+registry = Registry('cmathimpl')
+lower = registry.lower
```

Comment:
Until the new types are implemented, the use of `lower` and `overload` are creating duplicated entries. Should all these be commented out for now?

Reply:
Fixed in [93715b1](https://github.com/numba/numba/pull/9437/commits/93715b151e4350842fb227290eeda14d9f65c5ac)